TACC/protx
# Task/COOKS-57 add app route for reports

## Overview: ##

Add app routes for `reports` and `analysis`

## Related Jira tickets: ##

* [COOKS-57](https://jira.tacc.utexas.edu/browse/COOKS-57)

## Summary of Changes: ##
Add route for report and analysis
Lock dropdowns to demographic features, counties and 2019 when viewing report

## Testing Steps: ##

1. https://cep.dev/protx/report - > see report  (confirm that correct dropdowns are enabled as expected)
2. https://cep.dev/protx/analysis -> see analysis  (confirm that works are it did previously)
3. https://cep.dev/protx/foo -> see rerouted to https://cep.dev/protx/analysis
